x/tokenomics/keeper/token_logic_modules.go

```diff
@@ -681,6 +815,21 @@ func calculateGlobalPerClaimMintInflationFromSettlementAmount(settlementCoin sdk
 	return mintAmtCoin, *newMintAmtFloat
 }
 
+// stakeToMaxSettlementAmount calculates the max amount of uPOKT to that the supplier
```

Comment:
I find this function and comment confusing. 

- We're ONLY dealing with `%` here, no `uPOKT`
- Should `stakeTo` be `stakeShare`?
- Instead of `Amount`, should it be `Multiplier` like `1.2`?


----

After looking at the caller, I see that it's `maxClaimableAmt` being passed in, so maybe that should be the name of the parameter here.

---

Overall, I trust yo're doing the right thing but please revisit the naming of the function, the params, etc, so it's clear to a reader.